server/routerlicious/docker-compose.dev.yml

```diff
@@ -3,21 +3,28 @@ services:
     alfred:
         volumes:
             - .:/usr/src/server
+            - /usr/src/server/node_modules
     deli:
         volumes:
             - .:/usr/src/server
+            - /usr/src/server/node_modules
```

Comment:
`zookeeper` relies on `node-gyp-build` to build a binary. For this binary to work in a linux environment, it _must_ be built on the linux environment. Therefore, we cannot use the volume mounted node_modules when running locally.

Should I leave a comment about this in the code?

Reply:
Thanks for taking care of this, @znewton! I actually had a work item to later look into the `ZooKeeper` library, so this is great. Regarding the docker-compose changes, those are similar to what is needed when running RdKafka on a cross-platform environment (Docker Linux running on Mac, for example). @tanviraumi recommended not preventing volume mapping since it affects development agility, so what we do instead for the Kafka library is use Kafka-Node locally and RdKafka in kubernetes. Would it be possible to do something like `tryImport` here? https://github.com/microsoft/FluidFramework/blob/cd3c8a8f3419f0dd3f68fba4f73024dfad4a12e8/server/routerlicious/packages/services-ordering-rdkafka/src/tryImport.ts

@tanviraumi, any opinions?

Reply:
Correct! We need to keep our existing local dev story. So can we conditionally load it? I believe @GaryWilber added an undefined check everywhere, so even if we can't load it, rest of the system should just work

Reply:
I synced up with @znewton and looks like we can mount in a way that only affects zookeeper package. And rest of the local dev flow remains the same. With that, we can proceed with this approach

Reply:
Perfect! That's the best of both worlds 😄

Reply:
Updated docker-compose.dev to reduce the scope of this mount to zookeeper package, and added a note in the README. Thanks for the suggestions!